acid/templates/partials/status_list.html

```diff
@@ -42,7 +42,7 @@ <h6>
                             {% for job in buildset.jobs %}
                                 <tr>
                                     <td>
-                                        <a href="{{ job.log_url }}">{{ job.name }}</a> {% if not job.voting %} <span class="badge badge-secondary">non-voting</span>{% endif %}
+                                        <a {% if job.log_url %}href="{{ job.log_url }}"{% endif %}>{{ job.name }}</a> {% if not job.voting %} <span class="badge badge-secondary">non-voting</span>{% endif %}
```

Comment:
I think, when it's not linkable, it shouldn't be in <a> block

Reply:
Semantically wrong usage of anchor.
Anchor should always have `href` attribute.
It should be:
```html
{% if job.log_url %}
    <a href="{{ job.log_url }}">{{ job.name }}</a>
{% else %}
   <span>{{ job.name }}</span>
{% endif %}
```
or:
```html
<a href="{% if job.log_url %}{{ job.log_url }}{% else %}#{% endif %}">{{ job.name }}</a>
```